app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def post_comment_update
+    post.update(comments_counter: post.comments.count)
+  end
+end
```

Comment:
- [ ] As pointed out by the previous reviewer, please ensure that this belongs to the `author` instead of `user`